ods_ci/tests/Tests/0600__distributed_workloads/test-run-codeflare-tests.robot

```diff
@@ -9,10 +9,18 @@ Resource          ../../../tests/Resources/Page/DistributedWorkloads/Distributed
 
 
 *** Variables ***
-${CODEFLARE_RELEASE_ASSETS}     %{CODEFLARE_RELEASE_ASSETS=https://github.com/opendatahub-io/distributed-workloads/releases/latest/download}
-${NOTEBOOK_IMAGE}               %{NOTEBOOK_IMAGE_STREAM_NAME=quay.io/modh/odh-generic-data-science-notebook@sha256:9d7f80080a453bcf7dee01b986df9ee811ee74f6f433c601a8b67d283c160547}
-${NOTEBOOK_USER_NAME}           ${TEST_USER_3.USERNAME}
-${NOTEBOOK_USER_PASSWORD}       ${TEST_USER_3.PASSWORD}
+${CODEFLARE_RELEASE_ASSETS}         %{CODEFLARE_RELEASE_ASSETS=https://github.com/opendatahub-io/distributed-workloads/releases/latest/download}
+${NOTEBOOK_IMAGE}                   quay.io/modh/odh-generic-data-science-notebook@sha256:9d7f80080a453bcf7dee01b986df9ee811ee74f6f433c601a8b67d283c160547
+${NOTEBOOK_USER_NAME}               ${TEST_USER_3.USERNAME}
+${NOTEBOOK_USER_PASSWORD}           ${TEST_USER_3.PASSWORD}
+${CODEFLARE_TEST_RAY_IMAGE}         quay.io/rhoai/ray@sha256:859f5c41d41bad1935bce455ad3732dff9d4d4c342b7155a7cd23809e85698ab
```

Comment:
Is it possible to take the variable from `ods_ci/tests/Resources/Page/DistributedWorkloads/DistributedWorkloads.resource` ?

Reply:
I have migrated the codeflare test's variables and keywords into DistributedWorkloads.resource file, Please review : https://github.com/red-hat-data-services/ods-ci/pull/1657/commits/d9892709aad079b1adcef37ab3a1a6311664bcb1
@sutaakar Thank you for reviewing !!